### PR TITLE
AOS selinux bringup

### PR DESCRIPTION
**Pull request by LKomaryanskiy**

### CODE REVIEW

File: policy/modules/system/aos.fc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/usr/bin/aos_servicemanager             --  gen_context(system_u:object_r:aos_exec_t,s0)
+/usr/bin/aos_iamanager                  --  gen_context(system_u:object_r:aos_exec_t,s0)
+/usr/bin/aos_communicationmanager       --  gen_context(system_u:object_r:aos_exec_t,s0)
+/usr/bin/aos_updatemanager              --  gen_context(system_u:object_r:aos_exec_t,s0)
+/usr/bin/aos_vis                        --  gen_context(system_u:object_r:aos_exec_t,s0)
+
+/var/aos(/.*)?                              gen_context(system_u:object_r:aos_var_run_t,s0)
+
+/etc/aos(/.*)?                              gen_context(system_u:object_r:aos_conf_t,s0)
```
Comment (al1img) [inline policy/modules/system/aos.fc]: New line at the end.

File: policy/modules/services/dbus.te [hunk 2]
```diff
@@ -86,6 +86,7 @@ read_lnk_files_pattern(system_dbusd_t, dbusd_etc_t, dbusd_etc_t)
 manage_dirs_pattern(system_dbusd_t, system_dbusd_tmp_t, system_dbusd_tmp_t)
```
Comment (al1img) [inline policy/modules/services/dbus.te]: Commit message: can you elaborate more detail which issues this fix fixes?

File: policy/modules/system/aos.if [hunk 3]
```diff
@@ -57,3 +57,20 @@ interface(`aos_read_pid_files',`
...
+##	</summary>
+## </param>
+#
+interface(`files_getattr_aos_var_run_dir',`
+	gen_require(`
+		type aos_var_run_t;
+	')
+
+	allow $1 aos_var_run_t:dir getattr;
+')
```
Comment (al1img) [inline policy/modules/system/aos.if]: New line at the end.

Comment (LKomaryanskiy) [reply policy/modules/system/aos.if]: Added

File: policy/modules/services/dbus.te (hunk 2 above)
Comment (LKomaryanskiy) [reply policy/modules/services/dbus.te]: I've changed the commit and added more informative description.

File: policy/modules/system/aos.fc (hunk 1 above)
Comment (LKomaryanskiy) [reply policy/modules/system/aos.fc]: Added

**Review by al1img — APPROVED** [review]

**Pull request merged**